resource.go

```diff
@@ -28,6 +31,11 @@ type Resource interface {
 
 	UID() int64
 	GID() int64
+
+	ATime() time.Time
+	MTime() time.Time
+	CTime() time.Time
+	BTime() time.Time
```

Comment:
Well, it might be okay to follow the times library and just have `AccessTime`, `ModTime` and `ChangeTime`.